src/ocm/components/clusterWizard/ClusterPlatformIntegrationHint.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Alert, AlertVariant } from '@patternfly/react-core';
+import { Cluster } from '../../../common';
+import { useClusterSupportedPlatforms } from '../../hooks';
+
+type ClusterPlatformIntegrationHintProps = {
+  cluster: Cluster;
+};
+
+export const ClusterPlatformIntegrationHint: React.FC<ClusterPlatformIntegrationHintProps> = ({
+  cluster,
```

Comment:
```suggestion
  cluster,
```
Do we really need/want another component drilling down the cluster object? Why don't instead read it from the redux store?

Reply:
How can we read from redux store?

Reply:
You would use a selector for the cluster, in this case it's `selectCurrentClusterState`

Reply:
@jkilzi  if we use selectCurrentClusterState maybe the cluster is undefined and then the call to useClusterSupportedPlatforms(cluster.id) is not correct and I need to change the function useClusterSupportedPlatforms

Reply:
Finally, I change to pass the clusterId and platformType